support-frontend/assets/components/subscriptionCheckouts/address/postcodeFinder.jsx

```diff
@@ -78,20 +89,20 @@ class PostcodeFinder extends Component<PropTypes> {
     } = this.props;
     return (
       <div>
-        <ComposedInputWithButton
+        <InputWithButton
           error={error}
           label="Postcode"
           onClick={() => { fetchResults(postcode); }}
           id={id}
-          setValue={(val) => {
+          onChange={(val) => {
```

Comment:
Will the `onChange` callback here receive `val` as an argument or will it receive the `Event` (`e`)? If it's the `Event` we'll need to extract `e.target.value` in the body of the function and pass this value onto `setPostcode`/`onPostcodeUpdate`?

Reply:
Yes, this also needs to switch to e.target.value. Thanks again.